src/mono-runtimes/mono-runtimes.targets

```diff
@@ -107,6 +107,9 @@
     <_MonoUtility  Include="illinkanalyzer.exe" />
     <_MonoUtility  Include="mono-symbolicate.exe" />
     <_MonoUtility  Include="mkbundle.exe" />
+    <_MonoUtility  Include="mono-api-html.exe" />
+    <_MonoUtility  Include="mono-api-info.exe" />
```

Comment:
@alexischr likewise, do you want us to ship these in the installer?

Reply:
These two - no, they are only used in the build process. But pdb2mdb.exe (as I understand its use) should definitely be shipped in the installer.